src/panels/lovelace/types.ts

```diff
@@ -38,7 +38,10 @@ export interface LovelaceCard extends HTMLElement {
 }
 
 export interface LovelaceCardConstructor extends Constructor<LovelaceCard> {
-  getStubConfig?: (hass: HomeAssistant) => LovelaceCardConfig;
+  getStubConfig?: (
+    hass: HomeAssistant,
+    lovelaceCardConfig: LovelaceConfig
```

Comment:
It is called `lovelaceCardConfig` but is type `LovelaceConfig`?
```suggestion
    lovelaceConfig: LovelaceConfig
```

Reply:
oops good catch